### ISSUE TITLE
ETH Wallet Issue

### ISSUE THREAD

**Issue by jeremyhammon** [RainbowMiner/RainbowMiner#1439]
At least for me, it seems to API key of "freekey" has reached its max on ethplorer.  I have found 2 other exploreres that would seem to work (blockchair and blockcypher) but both are 1e18 multipliers and encounter the **bigint**  issue.

example for blockcypher:

```powershell
[PSCustomObject]@{symbol = "ETH";  match = "^0x";   rpc = "https://api.blockcypher.com/v1/eth/main/addrs/{w}";                    address = "address";                          balance = "balance";                         received = "total_received";                        divisor = 1e18}
```

**Comment by RainbowMiner** [comment]
Ok, I didn't know that there is a API max for ethplorer. I'll change it.

**Comment by RainbowMiner** [comment]
Done. I have switched the API provider to blockcypher - and solved the **bigint** problem (see issue #1432)
